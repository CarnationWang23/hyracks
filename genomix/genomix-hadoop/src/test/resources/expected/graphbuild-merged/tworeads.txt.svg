<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.26.3 (20100126.1600)
 -->
<!-- Title: tworeads_txt Pages: 1 -->
<svg width="644pt" height="321pt"
 viewBox="0.00 0.00 644.00 321.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph1" class="graph" transform="scale(1 1) rotate(0) translate(4 317)">
<title>tworeads_txt</title>
<polygon fill="white" stroke="white" points="-4,5 -4,-317 641,-317 641,5 -4,5"/>
<g id="graph2" class="cluster"><title>cluster_legend</title>
<polygon fill="none" stroke="black" points="49,-8 49,-209 211,-209 211,-8 49,-8"/>
<text text-anchor="middle" x="130" y="-192.4" font-family="Times Roman,serif" font-size="14.00">legend</text>
</g>
<g id="graph3" class="cluster"><title>cluster_1</title>
<polygon fill="none" stroke="black" points="8,-217 8,-305 376,-305 376,-217 8,-217"/>
</g>
<g id="graph4" class="cluster"><title>cluster_2</title>
<polygon fill="none" stroke="black" points="274,-121 274,-209 628,-209 628,-121 274,-121"/>
</g>
<!-- legend_0_0 -->
<g id="node2" class="node"><title>legend_0_0</title>
<ellipse fill="black" stroke="black" cx="59" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_0 -->
<g id="node3" class="node"><title>legend_1_0</title>
<ellipse fill="black" stroke="black" cx="201" cy="-157" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_0&#45;&gt;legend_1_0 -->
<g id="edge3" class="edge"><title>legend_0_0&#45;&gt;legend_1_0</title>
<path fill="none" stroke="#dd1e2f" d="M61.0074,-157C74.8673,-157 156.744,-157 188.46,-157"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="188.862,-160.5 198.862,-157 188.861,-153.5 188.862,-160.5"/>
<text text-anchor="middle" x="130" y="-162.4" font-family="Times Roman,serif" font-size="14.00">FF</text>
</g>
<!-- legend_0_1 -->
<g id="node5" class="node"><title>legend_0_1</title>
<ellipse fill="black" stroke="black" cx="59" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_1 -->
<g id="node6" class="node"><title>legend_1_1</title>
<ellipse fill="black" stroke="black" cx="201" cy="-116" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_1&#45;&gt;legend_1_1 -->
<g id="edge5" class="edge"><title>legend_0_1&#45;&gt;legend_1_1</title>
<path fill="none" stroke="#ebb035" d="M61.0074,-116C74.8673,-116 156.744,-116 188.46,-116"/>
<polygon fill="#ebb035" stroke="#ebb035" points="188.862,-119.5 198.862,-116 188.861,-112.5 188.862,-119.5"/>
<text text-anchor="middle" x="130" y="-121.4" font-family="Times Roman,serif" font-size="14.00">FR</text>
</g>
<!-- legend_0_2 -->
<g id="node8" class="node"><title>legend_0_2</title>
<ellipse fill="black" stroke="black" cx="59" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_2 -->
<g id="node9" class="node"><title>legend_1_2</title>
<ellipse fill="black" stroke="black" cx="201" cy="-75" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_2&#45;&gt;legend_1_2 -->
<g id="edge7" class="edge"><title>legend_0_2&#45;&gt;legend_1_2</title>
<path fill="none" stroke="#06a2cb" d="M61.0074,-75C74.8673,-75 156.744,-75 188.46,-75"/>
<polygon fill="#06a2cb" stroke="#06a2cb" points="188.862,-78.5001 198.862,-75 188.861,-71.5001 188.862,-78.5001"/>
<text text-anchor="middle" x="130" y="-80.4" font-family="Times Roman,serif" font-size="14.00">RF</text>
</g>
<!-- legend_0_3 -->
<g id="node11" class="node"><title>legend_0_3</title>
<ellipse fill="black" stroke="black" cx="59" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_1_3 -->
<g id="node12" class="node"><title>legend_1_3</title>
<ellipse fill="black" stroke="black" cx="201" cy="-34" rx="1.8" ry="1.8"/>
</g>
<!-- legend_0_3&#45;&gt;legend_1_3 -->
<g id="edge9" class="edge"><title>legend_0_3&#45;&gt;legend_1_3</title>
<path fill="none" stroke="#218559" d="M61.0074,-34C74.8673,-34 156.744,-34 188.46,-34"/>
<polygon fill="#218559" stroke="#218559" points="188.862,-37.5001 198.862,-34 188.861,-30.5001 188.862,-37.5001"/>
<text text-anchor="middle" x="130" y="-39.4" font-family="Times Roman,serif" font-size="14.00">RR</text>
</g>
<!-- 1,1 -->
<g id="node15" class="node"><title>1,1</title>
<ellipse fill="none" stroke="black" cx="59" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="39.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="37,-253 37,-267 82,-267 82,-253 37,-253"/>
<text text-anchor="start" x="40" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AATAGA</text>
<polygon fill="#218559" stroke="#218559" points="37,-239 37,-253 82,-253 82,-239 37,-239"/>
<text text-anchor="start" x="43" y="-243.667" font-family="Times Roman,serif" font-size="10.00">TCTATT</text>
</g>
<!-- 1,3 -->
<g id="node16" class="node"><title>1,3</title>
<ellipse fill="none" stroke="black" cx="201" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="181.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="179,-253 179,-267 224,-267 224,-253 179,-253"/>
<text text-anchor="start" x="185" y="-257.667" font-family="Times Roman,serif" font-size="10.00">TAGAA</text>
<polygon fill="#218559" stroke="#218559" points="179,-239 179,-253 224,-253 224,-239 179,-239"/>
<text text-anchor="start" x="187.5" y="-243.667" font-family="Times Roman,serif" font-size="10.00">TTCTA</text>
</g>
<!-- 1,1&#45;&gt;1,3 -->
<g id="edge25" class="edge"><title>1,1&#45;&gt;1,3</title>
<path fill="none" stroke="#dd1e2f" d="M101.605,-254.755C116.207,-254.208 132.729,-254.105 148.049,-254.448"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="148.326,-257.957 158.425,-254.756 148.534,-250.96 148.326,-257.957"/>
</g>
<!-- 1,3&#45;&gt;1,1 -->
<g id="edge31" class="edge"><title>1,3&#45;&gt;1,1</title>
<path fill="none" stroke="#218559" d="M158.425,-267.244C143.825,-267.792 127.305,-267.895 111.982,-267.553"/>
<polygon fill="#218559" stroke="#218559" points="111.704,-264.043 101.605,-267.245 111.497,-271.04 111.704,-264.043"/>
</g>
<!-- 1,4 -->
<g id="node17" class="node"><title>1,4</title>
<ellipse fill="none" stroke="black" cx="325" cy="-261" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-272.167" font-family="Times Roman,serif" font-size="10.00">1,4&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-253 303,-267 348,-267 348,-253 303,-253"/>
<text text-anchor="start" x="307.5" y="-257.667" font-family="Times Roman,serif" font-size="10.00">AGAAG</text>
<polygon fill="#218559" stroke="#218559" points="303,-239 303,-253 348,-253 348,-239 303,-239"/>
<text text-anchor="start" x="311" y="-243.667" font-family="Times Roman,serif" font-size="10.00">CTTCT</text>
</g>
<!-- 1,3&#45;&gt;1,4 -->
<g id="edge29" class="edge"><title>1,3&#45;&gt;1,4</title>
<path fill="none" stroke="#dd1e2f" d="M243.327,-254.562C252.601,-254.258 262.531,-254.176 272.159,-254.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="272.364,-257.822 282.443,-254.554 272.527,-250.823 272.364,-257.822"/>
</g>
<!-- 2,1 -->
<g id="node19" class="node"><title>2,1</title>
<ellipse fill="none" stroke="black" cx="325" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="305.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,1&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="303,-157 303,-171 348,-171 348,-157 303,-157"/>
<text text-anchor="start" x="307.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">AGAAG</text>
<polygon fill="#218559" stroke="#218559" points="303,-143 303,-157 348,-157 348,-143 303,-143"/>
<text text-anchor="start" x="311" y="-147.667" font-family="Times Roman,serif" font-size="10.00">CTTCT</text>
</g>
<!-- 1,3&#45;&gt;2,1 -->
<g id="edge27" class="edge"><title>1,3&#45;&gt;2,1</title>
<path fill="none" stroke="#dd1e2f" d="M234.345,-237.712C249.456,-227.157 264,-217 264,-217 264,-217 274.622,-207.945 287.002,-197.392"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="289.518,-199.846 294.857,-190.695 284.977,-194.519 289.518,-199.846"/>
</g>
<!-- 1,4&#45;&gt;1,3 -->
<g id="edge35" class="edge"><title>1,4&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M282.443,-267.446C273.158,-267.745 263.225,-267.824 253.602,-267.681"/>
<polygon fill="#218559" stroke="#218559" points="253.407,-264.175 243.327,-267.438 253.242,-271.173 253.407,-264.175"/>
</g>
<!-- 2,2 -->
<g id="node20" class="node"><title>2,2</title>
<ellipse fill="none" stroke="black" cx="449" cy="-165" rx="43.1335" ry="36.0624"/>
<text text-anchor="start" x="429.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,2&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="427,-157 427,-171 472,-171 472,-157 427,-157"/>
<text text-anchor="start" x="431.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">GAAGC</text>
<polygon fill="#218559" stroke="#218559" points="427,-143 427,-157 472,-157 472,-143 427,-143"/>
<text text-anchor="start" x="434" y="-147.667" font-family="Times Roman,serif" font-size="10.00">GCTTC</text>
</g>
<!-- 1,4&#45;&gt;2,2 -->
<g id="edge33" class="edge"><title>1,4&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M361.282,-241.419C375.287,-233.861 388,-227 388,-227 388,-227 400.484,-214.311 414.157,-200.414"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="416.837,-202.681 421.356,-193.098 411.847,-197.771 416.837,-202.681"/>
</g>
<!-- 2,1&#45;&gt;1,3 -->
<g id="edge15" class="edge"><title>2,1&#45;&gt;1,3</title>
<path fill="none" stroke="#218559" d="M293.015,-189.762C277.051,-202.122 257.587,-217.19 240.68,-230.28"/>
<polygon fill="#218559" stroke="#218559" points="238.376,-227.637 232.611,-236.527 242.661,-233.173 238.376,-227.637"/>
</g>
<!-- 2,1&#45;&gt;2,2 -->
<g id="edge13" class="edge"><title>2,1&#45;&gt;2,2</title>
<path fill="none" stroke="#dd1e2f" d="M367.327,-158.562C376.601,-158.258 386.531,-158.176 396.159,-158.316"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="396.364,-161.822 406.443,-158.554 396.527,-154.823 396.364,-161.822"/>
</g>
<!-- 2,2&#45;&gt;1,4 -->
<g id="edge19" class="edge"><title>2,2&#45;&gt;1,4</title>
<path fill="none" stroke="#218559" d="M417.015,-189.762C401.051,-202.122 381.587,-217.19 364.68,-230.28"/>
<polygon fill="#218559" stroke="#218559" points="362.376,-227.637 356.611,-236.527 366.661,-233.173 362.376,-227.637"/>
</g>
<!-- 2,2&#45;&gt;2,1 -->
<g id="edge21" class="edge"><title>2,2&#45;&gt;2,1</title>
<path fill="none" stroke="#218559" d="M406.443,-171.446C397.158,-171.745 387.225,-171.824 377.602,-171.681"/>
<polygon fill="#218559" stroke="#218559" points="377.407,-168.175 367.327,-171.438 377.242,-175.173 377.407,-168.175"/>
</g>
<!-- 2,3 -->
<g id="node21" class="node"><title>2,3</title>
<ellipse fill="none" stroke="black" cx="575" cy="-165" rx="44.0472" ry="36.0624"/>
<text text-anchor="start" x="555.5" y="-176.167" font-family="Times Roman,serif" font-size="10.00">2,3&#45;&#45;null</text>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="552,-157 552,-171 599,-171 599,-157 552,-157"/>
<text text-anchor="start" x="554.5" y="-161.667" font-family="Times Roman,serif" font-size="10.00">AAGCCC</text>
<polygon fill="#218559" stroke="#218559" points="552,-143 552,-157 599,-157 599,-143 552,-143"/>
<text text-anchor="start" x="555.5" y="-147.667" font-family="Times Roman,serif" font-size="10.00">GGGCTT</text>
</g>
<!-- 2,2&#45;&gt;2,3 -->
<g id="edge17" class="edge"><title>2,2&#45;&gt;2,3</title>
<path fill="none" stroke="#dd1e2f" d="M491.658,-158.573C501.09,-158.263 511.203,-158.176 521.018,-158.312"/>
<polygon fill="#dd1e2f" stroke="#dd1e2f" points="521.023,-161.813 531.097,-158.534 521.177,-154.814 521.023,-161.813"/>
</g>
<!-- 2,3&#45;&gt;2,2 -->
<g id="edge23" class="edge"><title>2,3&#45;&gt;2,2</title>
<path fill="none" stroke="#218559" d="M531.097,-171.466C521.607,-171.756 511.479,-171.824 501.692,-171.669"/>
<polygon fill="#218559" stroke="#218559" points="501.739,-168.169 491.658,-171.427 501.571,-175.167 501.739,-168.169"/>
</g>
</g>
</svg>
