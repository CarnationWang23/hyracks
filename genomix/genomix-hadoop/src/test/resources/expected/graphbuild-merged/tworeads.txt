((2,1)	[(2,2)]	[]	[]	[(1,3)]	AGAAG)	(null)
((2,2)	[(2,3)]	[]	[]	[(1,4),(2,1)]	GAAGC)	(null)
((2,3)	[]	[]	[]	[(2,2)]	AAGCCC)	(null)
((1,1)	[(1,3)]	[]	[]	[]	AATAGA)	(null)
((1,3)	[(2,1),(1,4)]	[]	[]	[(1,1)]	TAGAA)	(null)
((1,4)	[(2,2)]	[]	[]	[(1,3)]	AGAAG)	(null)
